lib/tasks/broadcasts.rake

```diff
@@ -0,0 +1,16 @@
+namespace :broadcasts do
+  desc "Send Welcome Notifications once a day"
+  task send_welcome_notification_flow: :environment do
+    # In order to prevent new users from receiving multiple welcome notifications in a day,
+    # a feature_live_date is required. The script will only be effective after feature_live_date
+    # and will ultimately be superseded by 7.days.ago when it's larger than feature_live_date.
+    next unless SiteConfig.welcome_notifications_live_at
+
+    notifications_live_at = SiteConfig.welcome_notifications_live_at
+    week_ago = 7.days.ago
+    latest_date = notifications_live_at > week_ago ? notifications_live_at : week_ago
```

Comment:
Do we plan on completely moving this code in the future or will this form of welcome notifications always be optional to turn on? If its the former might want to make a TODO to remove this code when we no longer need it.

Reply:
This will always be optional to turn on.